.github/BOTMETA.yml

```diff
@@ -415,11 +415,13 @@ files:
   $modules/monitoring/stackdriver.py: bwhaley
   $modules/monitoring/statusio_maintenance.py: bhcopeland
   $modules/monitoring/uptimerobot.py: nate-kingsley
-  $modules/monitoring/zabbix_group.py: cove eikef harrisongu
-  $modules/monitoring/zabbix_host.py: cove eikef harrisongu D3DeFi
-  $modules/monitoring/zabbix_hostmacro.py: cave eikef
-  $modules/monitoring/zabbix_maintenance.py: abulimov eikef
-  $modules/monitoring/zabbix_screen.py: cove eikef harrisongu
+  $modules/monitoring/zabbix/zabbix_group.py: cove eikef harrisongu
```

Comment:
Just for you information, the parent directory with your GitHub could be used:
```
$modules/monitoring/zabbix: eikef
$modules/monitoring/zabbix/zabbix_group.py: cove harrisongu
[...]
```

Reply:
Thank you for the pointer; pushed a commit to that effect :-)